gradle/shared-with-buildSrc/build-cache-configuration.settings.gradle.kts

```diff
@@ -27,6 +27,7 @@ val remoteCacheUsername = System.getProperty("gradle.cache.remote.username", "")
 val remoteCachePassword = System.getProperty("gradle.cache.remote.password", "")
 
 val isRemoteBuildCacheEnabled = remoteCacheUrl != null && gradle.startParameter.isBuildCacheEnabled && !gradle.startParameter.isOffline
+val disableLocalCache = System.getProperty("disableLocalCache")?.toBoolean() ?: false
```

Comment:
How about we choose a name similar to the other system properties? Something like `gradle.cache.local.disable`?

Reply:
I don't have a strong feeling about this, but I'd say not using `gradle.cache.local.disable`.

- `gradle.cache.local.disable` seems like a "data property" which carries some data.
- `disableLocalCache` looks like a switch, `-PignoreIncomingBuildReceipt=true -DdisableLocalCache=true` seem more natural.

Reply:
That is fine with me as well.